### ISSUE TITLE
Global time filter effects not saved for individual graphs

### ISSUE THREAD

**Issue by Calm-Rock** [SigNoz/signoz#3662]
## Bug description

When I view any metrics graph, I get an option to select the time interval for the graph. After selecting a particular time interval and saving it, it does not save the changes.

Here's a [video](https://www.loom.com/share/61cef37f86b049f7a22fdc1f67a17d57?sid=cd314c5c-b878-4182-b68d-ca13021d394a) for reference.

## Expected behavior

Ideally it should show me the graph with the time interval I selected.

## Version information 
* **Signoz version**: Cloud (SigNoz Demo)
* **Browser version**:
* **Your OS and version**: MacOs; Ventura
* **Your CPU Architecture**(ARM/Intel): ARM

**Comment by palashgdev** [comment]
@Rajat-Dabade let's disable the save for metrics application

**Comment by ankitnayan** [comment]
@Calm-Rock the `Panel Time Preference` seems to work when the panel is in `edit` mode. Can you check again?

> @Rajat-Dabade let's disable the save for metrics application

@palashgdev why disable for metrics application only? I don't see a usecase of saving a panel in `view` mode. We can remove `save` and `cancel` buttons for `view` mode.

@makeavish @srikanthccv do we want to save time range or anything in `view` mode of a panel?

**Comment by palashgdev** [comment]
> @Calm-Rock the `Panel Time Preference` seems to work when the panel is in `edit` mode. Can you check again?
> 
> > @Rajat-Dabade let's disable the save for metrics application
> 
> @palashgdev why disable for metrics application only? I don't see a usecase of saving a panel in `view` mode. We can remove `save` and `cancel` buttons for `view` mode.
> 
> @makeavish @srikanthccv do we want to save time range or anything in `view` mode of a panel?

yes it makes sense we can remove it completely as view mode is enabled

**Comment by palashgdev** [comment]
> makeavish @srikanthccv do we want to save time range or anything in `view` mode of a panel?

nothing over the backend side it stores over client side

**Comment by makeavish** [comment]
> @palashgdev why disable for metrics application only? I don't see a usecase of saving a panel in view mode. We can remove save and cancel buttons for view mode.

@ankitnayan Save and cancel is by design required for saving legend.
cc: @pranay01

**Comment by Calm-Rock** [comment]
> @Calm-Rock the `Panel Time Preference` seems to work when the panel is in `edit` mode. Can you check again?

@ankitnayan just checked. It works in the `edit` mode.
